aries-site/src/components/content/BodyText.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'grommet';
+
+// Body text size should be based on if its parent heading is an
+// h2, h3, etc.
+const TEXT_SIZE = {
+  1: 'large',
+  2: 'medium', // default font size
+  3: 'small',
+};
+
+export const BodyText = ({ children, level, ...rest }) => {
+  return (
+    <Text size={TEXT_SIZE[level]} {...rest}>
+      {children}
+    </Text>
```

Comment:
General question: Are there any tradeoffs/benefits of using <Text /> vs. <Paragraph />? Text renders as <span /> and  Paragraph as <p /> correct?

Reply:
Paragraph will automatically set a max-width depending on the font size. I am going to change BodyText to use Paragraph because these are paragraphs more than they are just text. However, for now I will apply the `fill` prop to the paragraph so that the paragraph can expand to the width of the box.